commander/src/commands/transaction/create/unlock.ts

```diff
@@ -0,0 +1,189 @@
+/*
+ * LiskHQ/lisk-commander
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import {
+	unlockToken,
+	utils as transactionUtils,
+} from '@liskhq/lisk-transactions';
+import {
+	isNumberString,
+	isValidFee,
+	isValidNonce,
+	validateAddress,
+} from '@liskhq/lisk-validator';
+import { flags as flagParser } from '@oclif/command';
+
+import BaseCommand from '../../../base';
+import { ValidationError } from '../../../utils/error';
+import { flags as commonFlags } from '../../../utils/flags';
+import { getNetworkIdentifierWithInput } from '../../../utils/network_identifier';
+import { getPassphraseFromPrompt } from '../../../utils/reader';
+
+interface RawAssetUnlock {
+	readonly delegateAddress: string;
+	readonly amount: string;
+	readonly unvoteHeight: number;
+}
+
+const createUnlockTransaction = (
+	nonce: string,
+	fee: string,
+	networkIdentifier: string,
+	unlockingObjects: ReadonlyArray<RawAssetUnlock>,
+	passphrase?: string,
+) =>
+	unlockToken({
+		nonce,
+		fee,
+		networkIdentifier,
+		passphrase,
+		unlockingObjects,
+	});
+
+interface Args {
+	readonly nonce: string;
+	readonly fee: string;
+}
+
+const splitInputs = (unlock: string) =>
+	unlock
+		.replace(/\n/g, ',')
+		.split(',')
+		.filter(Boolean)
```

Comment:
Does this mean we accept not just comma-separated values but a mix of comma-separated and per line entries both? I never saw it elsewhere in the commander. Is it regular practice?

Reply:
yes we were accepting comma separated values in vote transaction command as well.. yes one could enter to next line and enter the flags.. so that is the reason.

Reply:
But if one can enter to next line without completing value for a flag its an invalid user input. We should not strip it rather show an error to the user. 

This is a valid approaches: 

```
> transaction:create:unlock 1 1 --unlock=x,y,z
> transaction:create:unlock 1 1 --unlock=x,y,z --unlock=x,y,z 
```
or multiline 
```
> transaction:create:unlock 1 1 --unlock=x,y,z \
--unlock=x,y,z 
```

But this should be marked as an error. 

```
> transaction:create:unlock 1 1 --unlock=x,y \
,z 
```

Reply:
this is already taken care in `validateUnlocks` function

Reply:
My only concern is `.replace(/\n/g, ',')`, we should not this. For multiple arguments Oclif itself will handle and provide you an array.